src/engine-scripts/puppet/userMenuState.js

```diff
@@ -0,0 +1,5 @@
+const menuState = require( './menuState' );
+module.exports = async (page, hashtags) => {
+    const isClosed = hashtags.includes( '#userMenu-closed' );
+    menuState(page, '#p-personal-checkbox', isClosed);
```

Comment:
I think this might need either an await or return?

Reply:
Doe in follow up.